literalai/client.py

```diff
@@ -92,6 +93,15 @@ def instrument_llamaindex(self):
         """
         Instruments the Llama Index framework so that all RAG & LLM calls are logged to Literal AI.
         """
+
+        LLAMA_INDEX_REQUIREMENT = ["llama-index>=0.10.58"]
+
+        if not check_all_requirements(LLAMA_INDEX_REQUIREMENT):
+            raise Exception(
+                f"LlamaIndex instrumentation requirements not satisfied: {LLAMA_INDEX_REQUIREMENT}"
+            )
+        from literalai.instrumentation.llamaindex import instrument_llamaindex
+
```

Comment:
did you try that everything is still working? we could alias the import to avoid functions with the exact same name

Reply:
I didn't try to instrument llama index, just the regular literal client.
I'll run a couple of cookbooks and alias the import that's more neat.

Reply:
Works well: 
![Uploading image.png…]()